web-common/src/main/java/org/openmrs/eip/web/contoller/BaseRestController.java

```diff
@@ -52,6 +53,31 @@ public Object doGet(Long id) {
 		    "jpa:" + getName() + "?query=SELECT c FROM " + getName() + " c WHERE c.id = " + id, null, getClazz());
 	}
 	
+	public Map<String, Object> getByDateCreated(SenderSearch search) {
+
+	        Map<String, Object> results = new HashMap<String, Object>(2);
+	        List<Object> items;
+
+	        if (search.getStartDate().isBlank() || search.getEndDate().isBlank()) {
```

Comment:
I think you should support searches by either startDate or endDate i.e. items before or after a given date

Reply:
Done